tslint.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Did you build this to roughly match our eslint config rules?

Reply:
Hey @jaxesn the `tslint.json` here was taken from the work done in the UI Kit repo (https://github.com/dcos-labs/ui-kit). 

But to answer your question, yes I used our eslint as a reference point for the rules that tslint support.

A lot of those rules aren't supported by tslint or are a bit different so for that reason, we have to use https://github.com/buzinas/tslint-eslint-rules but even though some rules aren't present which I believe should be fine.